examples/include/data-utils.hpp

```diff
@@ -4,6 +4,8 @@
 
 #pragma once
 
+#include <../../include/dr/detail/format_shim.hpp>
```

Comment:
will `#include <dr/detail/format_shim.hpp>` not work?

Reply:
No, and I don't really understand why not.

Reply:
Build with `make VERBOSE=1` and look at the compile line. Does it have a `-I` that points to the include directory? Maybe it is included in an example that does not otherwise use DR and it is not being added. It is OK to add DR as a link library even if it only uses this part.